src/_pytest/fixtures.py

```diff
@@ -1060,14 +1066,14 @@ def _parse_fixture_args(callable_or_scope, *args, **kwargs):
 
 
 def fixture(
-    callable_or_scope=None,
+    callable_or_scope: Union[Callable[..., Any], str] = None,
     *args,
```

Comment:
you'll also need `*args: Any`  such that mypy will consider this function as typed